AdminPanelService/AdminPanel.API/Controllers/CarModelController.cs

```diff
@@ -3,10 +3,12 @@
 using AdminPanel.BLL.CQS.CatalogueService.Commands.CarModelCommands.UpdateCarModel;
 using AdminPanel.BLL.CQS.CatalogueService.Queries.CarModelQueries.GetCarModelById;
 using AdminPanel.BLL.CQS.CatalogueService.Queries.CarModelQueries.GetCarModelsInRange;
+using Microsoft.AspNetCore.Authorization;
 
 namespace AdminPanel.API.Controllers;
 
 [ApiController]
+[Authorize("use:dashboard")]
```

Comment:
Here and in other files would it be better to use contant instead of the same string everywhere?